pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -890,3 +890,190 @@ func TestVolumeBinding(t *testing.T) {
 		})
 	}
 }
+
+func TestIsSchedulableAfterCSIDriverChange(t *testing.T) {
+	trueVar := true
+	falseVar := false
+	table := []struct {
+		name   string
+		pod    *v1.Pod
+		newObj interface{}
+		oldObj interface{}
+		err    bool
+		expect framework.QueueingHint
+	}{
+		{
+			name: "pod has no CSIDriver",
+			pod:  makePod("pod-a").Pod,
+			newObj: &storagev1.CSIDriver{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test1",
+				},
+				Spec: storagev1.CSIDriverSpec{
+					StorageCapacity: &trueVar,
+				},
+			},
+			oldObj: &storagev1.CSIDriver{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test1",
+				},
+			},
+			err:    false,
+			expect: framework.QueueSkip,
+		},
+		{
+			name:   "type conversion error",
+			pod:    makePod("pod-a").Pod,
+			newObj: new(struct{}),
+			oldObj: new(struct{}),
+			err:    true,
+			expect: framework.Queue,
+		},
+		{
+			name: "driver name in pod and csidriver name are different.",
+			pod:  makePod("pod-a").withCSI("test1").Pod,
+			newObj: &storagev1.CSIDriver{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test2",
+				},
+				Spec: storagev1.CSIDriverSpec{
+					StorageCapacity: &trueVar,
```

Comment:
So, we have to test these two behaviours:
- [First] When a driver name in the pod and a csidriver name are different, it returns Skip.
- [Second] When a csidriver's storage capacity is still enabled after the update, it returns Skip.

And, you want to make sure the first one with this test case. Then, it's better to change like ↓ so that this test case returns Skip for the first point, not for the second point.

```suggestion
					StorageCapacity: ptr.To(false),
```

Then, if there was a bug for the first point, this test case would fail.

Reply:
I fix it as per your comment.
[f46d8e9](https://github.com/kubernetes/kubernetes/pull/125171/commits/f46d8e9c2abbfad2e46270864ed150aa46763aa7)